:memo: `justify-content: space-evenly` is only supported by Firefox

borisschapira opened the pull request:
Adds the info about Firefox being the only browser that supports `justify-content: space-evenly` into the list of bugs for the Flexible Box Layout Module.

Schweinepriester commented:
One could argue this should be covered by https://github.com/Fyrd/caniuse/issues/2537 (or) on its own.

Fyrd commented:
@borisschapira Your entry is missing the `"description":` key.

borisschapira commented:
Thanks @Fyrd, I apologize. I've added one.

cvrebert commented:
@borisschapira Doesn't seem to be showing up. Perhaps you didn't push the commit?

cvrebert commented on features-json/flexbox.json:
```diff
@@ -69,6 +69,9 @@
       "description":"Safari 10 and below uses min/max width/height declarations for actually rendering the size of flex items, but it ignores those values when calculating how many items should be on a single line of a multi-line flex container. Instead, it simply uses the item's flex-basis value, or its width if the flex basis is set to auto. [see bug](https://bugs.webkit.org/show_bug.cgi?id=136 ...
+    },
+    {
+      "The space-evenly value of the justify-content property, as described in the [CSS Box Alignement Module Level 3](https://www.w3.org/TR/css-align-3/#distribution-values) is only supported by Firefox."
```
"Alignment", not "Alignement" [sic]

borisschapira commented:
Ok… I've understood my PR was missing a description… (you can laugh).
I've added a description key in the file and fixed the typo on "Alignment" (thks for having spotted it).

SelenIT commented:
In fact, Firefox is (currently) the only browser that supports `space-evenly` for _Flexbox_ layout. For Grid layout only, Chrome and Safari support it as well. Such a partial support was their bug that has already been fixed:
https://bugs.chromium.org/p/chromium/issues/detail?id=708121#c13
https://bugs.webkit.org/show_bug.cgi?id=170445

Fyrd commented:
Thanks @borisschapira ! Sorry for not being more clear on what was needed. :D
